src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/IndexRequest.cs

```diff
@@ -13,6 +13,7 @@ public class IndexRequest : IAuthorizationContextModel
         public string AccountLegalEntityHashedId { get; set; }
 
         public string StartMonthYear { get; set; }
+        public DeliveryModel? DeliveryModel { get; set; }
```

Comment:
I don't think DeliveryModel needs to be here

Reply:
OK, removed